### ISSUE TITLE
preferred name for City of London

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#1364]
In record `1158857257` the `eng_x_preferred` name is `City`, which is not very descriptive.

Can we please change this to `City of London`?

Looking at the other names, `The City` can probably be moved to colloquial?

**Comment by stepps00** [comment]
Agreed that this name is not very descriptive. I'm in the process of updating lower-level admin records in the UK, I will note this and add a fix to an upcoming PR.

**Comment by stepps00** [comment]
Related: https://github.com/whosonfirst-data/whosonfirst-data/issues/1367#issuecomment-435144511

**Comment by missinglink** [comment]
Looks good, I would still prefer we didn't have `City` in `name:eng_x_variant` because it isn't the given name of a place.

Maybe we can move it to a meta-data property instead such as `wof:placetype_local`?

"The City" is correct as colloquial:
> a term for the City of London, the historic core of London; also used to refer to the British financial services sector

**Comment by nvkelso** [comment]
Eh, variant  names are just a random grab bag, I think both `City` and `The City` are both valid variants of the preferred `City of London`. Agree that it can also be colloquial.

> Maybe we can move it to a meta-data property instead such as wof:placetype_local?

No, that's confusing names and placetypes.

**Comment by stepps00** [comment]
Noting here that the first phase of UK admin work was merged recently. This included a change to the "The City" record (see it [here](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/master/data/115/885/725/7/1158857257.geojson)) that updated the English colloquial, variant, and preferred names for the record.

The name "City" was sourced from the Open London data that we used last year when updating London neighbourhoods, so I'd like to store it in the variant namespace. 

There is another round of UK locality work coming up, so I'm happy to revisit this record if something still looks off.
